src/main.js

```diff
@@ -57,7 +57,7 @@ const screens = {
             debug: "debug",
             //Example of custom routing function
             next: scene => {
-                scene.navigate("narrative");
+                scene.navigate("shop");
```

Comment:
We probably don't want to leave this in for now. Might be time to set up a debug screen. Maybe as part of whatever is next?

Reply:
Yeah, this needs to come out.

Reply:
b283938